source/_components/switch.doorbird.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: "DoorBird Switch"
+description: "Instructions how to integrate DoorBird video doorbell relays into Home Assistant."
+date: 2017-08-06 11:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: doorbird.png
+ha_category: Switch
+ha_release: "0.50"
```

Comment:
Release will need to be updated before merge. Most likely to 0.52 since 0.51 is rather imminent.